.github/workflows/build.yml

```diff
@@ -127,4 +129,96 @@ jobs:
       uses: actions/upload-artifact@v3
       with:
         name: Test Results (${{ matrix.os }} ${{ matrix.configuration }} ${{ matrix.projectloadstyle }})
-        path: test-results/*
\ No newline at end of file
+        path: test-results/*
+        
+  sign:
```

Comment:
this should be a separate workflow that only builds on main or release/ and is triggered by a successful build there

Reply:
Yes, I'll separate them out.